shared/aggregation/aggregation_test.go

```diff
@@ -0,0 +1,54 @@
+package aggregation
+
+import (
+	"math/rand"
+	"time"
+
+	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	"github.com/prysmaticlabs/go-bitfield"
+	"github.com/prysmaticlabs/prysm/shared/bls"
+)
+
+func bitlistWithAllBitsSet(length uint64) bitfield.Bitlist {
+	b := bitfield.NewBitlist(length)
+	for i := uint64(0); i < length; i++ {
+		b.SetBitAt(i, true)
+	}
+	return b
+}
+
+func bitlistsWithSingleBitSet(n, length uint64) []bitfield.Bitlist {
+	lists := make([]bitfield.Bitlist, n)
+	for i := uint64(0); i < n; i++ {
+		b := bitfield.NewBitlist(length)
+		b.SetBitAt(i%length, true)
+		lists[i] = b
+	}
+	return lists
+}
+
+func bitlistsWithMultipleBitSet(n, length, count uint64) []bitfield.Bitlist {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
This makes flaky test failures difficult to reproduce. I recommend a constant seed value or t.Log of the value used as the seed, in case you need to reproduce a test failure locally.

Reply:
Thanks, I am logging seed now.